lib/Doctrine/Common/Collections/ExpressionBuilder.php

```diff
@@ -42,7 +42,7 @@ class ExpressionBuilder
      */
     public function andX($x = null)
     {
-        return new CompositeExpression(CompositeExpression::TYPE_AND, func_get_args());
+        return new CompositeExpression(CompositeExpression::TYPE_AND, is_array($x) ? $x : func_get_args());
```

Comment:
This should be changed to just accept `$x`, and the method signature should become `...$x`